source/cloud-native-platform/path-to-live/shutter.html.md.erb

```diff
@@ -146,6 +146,10 @@ CNAME records are much easier to shutter, for each record an equivalent shutter
 
 As long as there is a matching shutter config for the DNS record you can update the shutter value to true to enable a shutter.
 
+## Shuttering all CFT services
```

Comment:
Do we label this as CFT specific or shutter all services within a zone?

Reply:
This relates to teams shuttering all CFT services specifically. This has been required for certain releases and this is just so teams can do this themselves going forward. Think there's further improvements to this planned so this will be updated again once those have been implemented